src/java.base/share/classes/java/util/TreeMap.java

```diff
@@ -420,10 +420,9 @@ else if (cmp > 0)
     }
 
     /**
-     * Gets the entry corresponding to the specified key; if no such entry
-     * exists, returns the entry for the least key greater than the specified
-     * key; if no such entry exists (i.e., the greatest key in the Tree is less
-     * than the specified key), returns {@code null}.
+     * Returns the entry for the least key greater than or equal to the specified key;
+     * if no such entry exists (i.e. the specified key is greater than any key in the tree),
```

Comment:
I'd suggest changing "the specified key is greater than any key in the tree" → "the specified key is greater than any key in the tree, or the tree is empty"; same thing for `getFloorEntry()`.

Reply:
It's definitely inferrable (and the phrasing was from the original, so not your fault) but it's good to eliminate any ambiguity, however minimal, for a diverse public readership. Thanks.